vue-client/public/index.html

```diff
@@ -10,11 +10,12 @@
     <meta property="og:title" content="Libris katalogisering">
     <meta property="og:site_name" content="Libris katalogisering">
     <meta property="og:description" content="Cataloguing interface for the Swedish National Union Catalogue">
-    <meta property="og:image" content="https://libris<%= process.env.VUE_APP_ENV_LABEL ? '-' : '' %><%= process.env.VUE_APP_ENV_LABEL %>.kb.se<%= BASE_URL %>opengraph.png">
+    <meta property="og:image" content="https://libris.kb.se<%= BASE_URL %>opengraph.png">
```

Comment:
With these changes, this value cannot be known at build-time/template processing time.
Don't know how crucial it is to have the right url pointed out from test ends here. With current implementation it was sacrificed.